tardis/montecarlo/montecarlo_numba/base.py

```diff
@@ -227,13 +227,23 @@ def montecarlo_main_loop(
         vpacket_collection = vpacket_collections[i]
         rpacket_tracker = rpacket_trackers[i]
 
+        if montecarlo_configuration.RPACKET_TRACKING == 0:
```

Comment:
This project should be done purely externally from the internal workings of tardis.  Please do not change the default behavior if it is not part of the project.